rules/windows/process_creation/proc_creation_win_nslookup_poweshell_download.yml

```diff
@@ -0,0 +1,27 @@
+title: Nslookup PowerShell Download
```

Comment:
same as ref than id 72671447-4352-4413-bb91-b85569687135 (proc_creation_win_nslookup_pwsh_download_cradle.yml).
You can make an update.

Reply:
Will do